egs2/sdsv21/spk1/local/convert_trial.py

```diff
@@ -0,0 +1,54 @@
+# convert_trial.py
+
+import argparse
+import os
+import sys
+
+
+def main(args):
+    with open(args.trial, "r") as f:
+        lines_trial_org = f.readlines()
+    with open(args.scp, "r") as f:
+        lines_scp = f.readlines()
+
+    scp_dict = dict()
+    for scp in lines_scp:
+        utt_id, path = scp.strip().split(" ")
+        scp_dict[utt_id] = path
+
+    with open(os.path.join(args.out, "trial.scp"), "w") as f_trial, open(
+        os.path.join(args.out, "trial2.scp"), "w"
+    ) as f_trial2, open(os.path.join(args.out, "trial_label"), "w") as f_label:
+        for tr in lines_trial_org:
+            label, utt1, utt2 = tr.strip().split(" ")
+            utt1 = utt1[:-4]
```

Comment:
os.path.splitext(utt1)[0] ?

Reply:
I see, I had not changed this file from the Voxceleb recipe and hadn't noticed that
I assume this could be better, based on your comment:
`label, utt1, utt2 = tr.strip().split(" ")`
`utt1 = os.path.splitext(utt1)[0]`